### ISSUE TITLE
Unable to delete with DJIMediaManager.deleteMedia

### ISSUE THREAD

**Issue by yishus** [dji-sdk/Mobile-SDK-iOS#68]
DJIMediaManager deleteMedia method returns
```
Error Domain=DJISDKErrorDomain Code=-1008 "There was an error executing the command.(code:-1008)" UserInfo={NSLocalizedDescription=There was an error executing the command.(code:-1008)}
```

Have set the camera mode to DJICameraMode.mediaDownload, pass in [DJIMedia]. 

iOS SDK version 3.4.1 and version 4 (alpha)
Mavic Pro
iOS 10.1
iPhone 6
Xcode 8.1

**Comment by oliverou** [comment]
Hi @yishus, I saw you still use iOS SDK 3.4.1 for testing, we have published the new iOS SDK 3.5 recently, could you download it and check if the issue still exist?  You can get the latest SDK from: <https://developer.dji.com/mobile-sdk/downloads/>

**Comment by yishus** [comment]
Hi @oliverou, updated the SDK and realise it still happens. But have realised the problem only happens if I delete the media soon after downloading the thumbnail with `fetchThumbnail(completion block: DJISDK.DJICompletionBlock? = nil)`

For now, I'm putting in a delay if this happens.
